gsi/proxy/proxy_ssl/source/library/proxycertinfo.h

```diff
@@ -154,6 +154,9 @@ ASN1_METHOD * PROXYCERTINFO_asn1_meth();
 
 GLOBUS_DEPRECATED(
 PROXYCERTINFO * PROXYCERTINFO_dup(
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
+    const
+#endif
     PROXYCERTINFO *                     cert_info));
```

Comment:
@msalle:
Sorry, but why do we need a `const` here (and in the file below) with OpenSSL 3.0.0? I can't find the `PROXYCERTINFO_dup()` function in [OpenSSL's repo](https://github.com/openssl/openssl) nor did I find the `PROXYCERTINFO` type there.

Reply:
They are defined around https://github.com/gridcf/gct/blob/master/gsi/proxy/proxy_ssl/source/library/proxycertinfo.h#L136 via a bunch of macros in https://github.com/openssl/openssl/blob/master/include/openssl/asn1.h.in
So in this case for the name `PROXYCERTINFO`. The `*_dup()` function now wants a `const`, see https://github.com/openssl/openssl/blob/master/include/openssl/asn1.h.in#L289 and commit https://github.com/openssl/openssl/commit/9fdcc21fdc9d148f78d9cd5be34030f38cc45812

Reply:
Ok, thanks for the explanation.